exercises/rotational-cipher/example.py

```diff
@@ -0,0 +1,12 @@
+def rotational_cipher(message, key):
+    import string
+    alpha_lower = list(string.ascii_lowercase)
+    alpha_upper = list(string.ascii_uppercase)
+    coded_message = ""
+    for char in list(message):
```

Comment:
The python strings already support the sequence type methods, so you don't need to convert them to lists to use `for` or `index()`.